vue-components/src/components/TextInput.vue

```diff
@@ -174,7 +125,7 @@ $base: ".wikit-TextInput";
 	}
 
 	& #{$base}__label {
-		color: $wikit-TextInput-input-color;
+		color: $wikit-TextInput-label-font-color;
```

Comment:
This has changed but I can't find out why.

Reply:
This should have used the `$wikit-TextInput-label-font-color` to begin with, and only looked correct by chance. The fact that we're using the wrong token here surfaced when I extracted the Input element and its styles, and removed all `$wikit-TextInput-input-...` tokens, which shouldn't have been referenced anywhere anymore at that point.

I can extract it into a separate commit if it looks confusing.

Reply:
no, just had a question about it, it's too small to be extracted. Thanks!